packages/jest-core/src/collectHandles.ts

```diff
@@ -52,7 +52,11 @@ export default function collectHandles(): () => Array<Error> {
       _triggerAsyncId,
       resource: {} | NodeJS.Timeout,
     ) {
-      if (type === 'PROMISE' || type === 'TIMERWRAP') {
+      if (
+        type === 'PROMISE' ||
+        type === 'TIMERWRAP' ||
+        type === 'ELDHISTOGRAM'
```

Comment:
just to verify, if you remove this new condition, the test you've added fails, right?

Reply:
I verified now, and the test does indeed fail without this change 👍 

![image](https://user-images.githubusercontent.com/1404810/90691314-6c0b0780-e273-11ea-80e5-d556419b4402.png)